amf-api-contract/shared/src/main/scala/amf/apicontract/internal/metamodel/domain/bindings/IBMMQBindingModel.scala

```diff
@@ -0,0 +1,62 @@
+package amf.apicontract.internal.metamodel.domain.bindings
+
+import amf.apicontract.client.scala.model.domain.bindings.ibmmq.IBMMQMessageBinding
+import amf.core.client.scala.model.domain.AmfObject
+import amf.core.client.scala.vocabulary.Namespace.ApiBinding
+import amf.core.client.scala.vocabulary.ValueType
+import amf.core.internal.metamodel.Field
+import amf.core.internal.metamodel.Type.{Int, Str}
+import amf.core.internal.metamodel.domain.{ModelDoc, ModelVocabularies}
+
+object IBMMQMessageBindingModel extends MessageBindingModel with BindingVersion {
+  val MessageType: Field =
+    Field(
+      Str,
+      ApiBinding + "messageType",
+      ModelDoc(ModelVocabularies.ApiBinding, "type", "The type of the message")
+    )
+
+  val Headers: Field =
+    Field(
+      Str,
+      ApiBinding + "headers",
+      ModelDoc(
+        ModelVocabularies.ApiBinding,
+        "headers",
+        "Defines the IBM MQ message headers to include with this message. More than one header can be specified as a comma separated list."
+      )
+    )
+
+  val Description: Field =
+    Field(
+      Str,
+      ApiBinding + "description",
```

Comment:
I think we already have a description propertyTerm here. Maybe we can reuse this one. Maybe ask Pope?

Reply:
done, we're now using the core description field